src/bpmncwpverify/cwp.py

```diff
@@ -0,0 +1,247 @@
+from typing import Dict, List, Optional
```

Comment:
BTW, we need a similar naming convention for the CWP states as we have for the BPMN elements. What naming convention are we currently using here?

Reply:
That's a great question, I think we are currently using a cleanup function like we were before. I can switch it so that it uses the ids if you think that would be good. But it looks like we need a cleanup method for the edges' expression attribute in order for the expressions to be consistent:

`    def cleanup_expression(self, expression: str) -> None:
        expression = re.sub(r"&amp;", "&", expression)
        expression = re.sub(r"&lt;", "<", expression)
        expression = re.sub(r"&gt;", ">", expression)

        expression = re.sub(r"</?div>", "", expression)
        expression = re.sub(r"<br>", " ", expression)

        expression = re.sub(r"\s*(==|!=|&&|\|\|)\s*", r" \1 ", expression)

        expression = re.sub(r"\s+", " ", expression)

        self.expression = expression.strip()
`